jme3-alloc/src/main/java/com/jme3/alloc/util/NativeBufferUtils.java

```diff
@@ -0,0 +1,116 @@
+package com.jme3.alloc.util;
+
+import java.io.IOException;
+import java.nio.Buffer;
+import com.jme3.alloc.util.loader.NativeImageLoader;
+
+/**
+ * A jvm low-level implementation to the native GNU-Standard-Library [stdlib].
+ * 
+ * @author pavl_g
+ */
+public final class NativeBufferUtils {
+
+    static {
+        try {
+            /* extracts and loads the system specific library */
+            NativeImageLoader.loadLibrary();
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+    }
+
+    private NativeBufferUtils() {
+    }
+
+    /**
+     * Allocates a buffer of size bytes.
+     * 
+     * @param size the size of the newly created buffer in bytes (max = 32-bit or 4 bytes)
+     * @return a new java-nio buffer object
+     */
+    public static native Buffer memoryAlloc(final long size);
```

Comment:
It will be nice to document that it uses: `{@link NativeBufferUtils#malloc(long)}`

Reply:
This is the jvm version for the GNU stdlib `void* malloc(size_t)` function, but i am trying to follow the java naming conventions (google java coding style).....let me know if you would like to break this rule and use the GNU style here.

Reply:
Note, I do not mean to rename the method, just was saying that add a Javadoc on this method and mention the underlying it uses malloc.

Reply:
Oh okay, i misunderstood this sorry.